app/src/test/java/org/simple/clinic/summary/PatientSummaryScreenControllerTest.kt

```diff
@@ -1447,4 +1341,53 @@ class PatientSummaryScreenControllerTest {
         prescribedDrugsChanged = false
     )
   }
+
+  private fun setupControllerWithScreenCreated(
+      openIntention: OpenIntention,
+      patientUuid: UUID = this.patientUuid,
+      screenCreatedTimestamp: Instant = Instant.now(utcClock),
+      numberOfBpPlaceholders: Int = 0,
+      numberOfBpsToDisplay: Int = this.bpDisplayLimit,
+      bpEditableDuration: Duration = Duration.ofMinutes(60)
+  ) {
+    val config = PatientSummaryConfig(numberOfBpPlaceholders, numberOfBpsToDisplay, bpEditableDuration)
+    createController(config)
+
+    configSubject.onNext(config)
```

Comment:
This can be replaced with `setupControllerWithoutScreenCreated`.

Reply:
Done.